src/MenuItem/index.js

```diff
@@ -0,0 +1,31 @@
+import './styles.scss';
+
+const MenuItem = ({menuItemData}) => {
+    return (
+        <div className="col-12 col-md-4 col-lg-2">
+            <div className="card shadow mb-3 border-dark">
+                <div className="card-body">
+                    <h4 className="card-title">{menuItemData.foodName}</h4>
+                    <ul className="list-unstyled">
+                        <li className="list-group-item  rounded fw-bold"><span class='calories badge'>Calories: {menuItemData.calories}</span></li>
+                        {menuItemData.foodType &&
+                            <li className="list-group-item rounded fw-bold"><span className='type badge'>Type: {menuItemData.foodType}</span></li>
+                        }
+                        {!menuItemData.foodType &&
+                            <li className="d-none"></li>
+                        }
+                        {menuItemData.sideItem &&
+                            <li className="list-group-item rounded fw-bold"><span className='side badge'>Side</span></li>
+                        }
+                        {!menuItemData.sideItem &&
+                            <li className="d-none"></li>
+                        }
+                    </ul>
+                    <p className="fw-bold">£ {menuItemData.price} </p>
```

Comment:
Some of the items are not to 2 decimal places, think we need something in here to ensure all prices are in full i.e Quarter Pound with Cheese is currently £12.1 and it should be £12.10